libs/ui/src/print_element.tsx

```diff
@@ -58,9 +59,8 @@ async function printElementWhenReadyHelper<PrintResult>(
       await printAndTeardown();
     }
 
-    ReactDom.render(
-      <PrintOnly>{elementWithOnReadyCallback(onElementReady)}</PrintOnly>,
-      printRoot
+    reactPrintRoot.render(
+      <PrintOnly>{elementWithOnReadyCallback(onElementReady)}</PrintOnly>
```

Comment:
The main change to non-test files is this change in how `react` is rendered at roots.